README.md

```diff
@@ -290,29 +302,40 @@ For example, to include line numbers for a `assertEquals()` function call, repla
 
 Example -- Asserts with and without line numbers
 
-    #! /bin/sh
-    # file: examples/lineno_test.sh
-    
-    testLineNo()
-    {
-        # this assert will have line numbers included (e.g. "ASSERT:[123] ...")
-        echo "ae: ${_ASSERT_EQUALS_}"
-        ${_ASSERT_EQUALS_} 'not equal' 1 2
-        
-        # this assert will not have line numbers included (e.g. "ASSERT: ...")
-        assertEquals 'not equal' 1 2
-    }
+```sh
+#! /bin/sh
+# file: examples/lineno_test.sh
+
+testLineNo()
+{
+    # this assert will have line numbers included (e.g. "ASSERT:[123] ...")
+    echo "ae: ${_ASSERT_EQUALS_}"
+    ${_ASSERT_EQUALS_} 'not equal' 1 2
     
-    # load shunit2
-    . ../src/shell/shunit2
+    # this assert will not have line numbers included (e.g. "ASSERT: ...")
+    assertEquals 'not equal' 1 2
+}
+
+# load shunit2
+. ../src/shell/shunit2
+```
 
 Notes:
 
-1. Due to how shell parses command-line arguments, all strings used with macros should be quoted twice. Namely, single-quotes must be converted to single-double-quotes, and vice-versa. If the string being passed is absolutely for sure not empty, the extra quoting is not necessary.<br /><br />Normal `assertEquals` call.<br/><br/>`assertEquals 'some message' 'x' ''`<br/><br/>Macro `_ASSERT_EQUALS_` call. Note the extra quoting around the _message_ and the _null_ value.
+1. Due to how shell parses command-line arguments, all strings used with macros should be quoted twice. Namely, single-quotes must be converted to single-double-quotes, and vice-versa. If the string being passed is absolutely for sure not empty, the extra quoting is not necessary.
+   Normal `assertEquals` call.
```

Comment:
The original rendering had an empty line above this one. Is it possible to recreate it here?